Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 King's College London
+# Created by the Software Development Team <http://soft-dev.org/>
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0>, or the MIT license <LICENSE-MIT
+# or http://opensource.org/licenses/MIT>, or the UPL-1.0 license
+# <http://opensource.org/licenses/UPL> at your option. This file may not be
+# copied, modified, or distributed except according to those terms.
+
+FETCH=wget
+
+RENAISSANCE_V=0.9.0
+RENAISSANCE_JAR=renaissance-gpl-${RENAISSANCE_V}.jar
+
+GRAALCE_V=1.0.0-rc16
+GRAALCE_DIR=graalvm-ce-${GRAALCE_V}
+GRAALCE_TGZ=graalvm-ce-${GRAALCE_V}-linux-amd64.tar.gz
+
+all: setup
+
+setup: ${RENAISSANCE_JAR} ${GRAALCE_DIR}
+
+${RENAISSANCE_JAR}:
+	echo ${RENAISSANCE_JAR}
+	${FETCH} https://github.com/renaissance-benchmarks/renaissance/releases/download/v${RENAISSANCE_V}/${RENAISSANCE_JAR}
+
+${GRAALCE_TGZ}:
+	${FETCH} https://github.com/oracle/graal/releases/download/vm-${GRAALCE_V}/${GRAALCE_TGZ}
+
+${GRAALCE_DIR}: ${GRAALCE_TGZ}
+	tar xfz ${GRAALCE_TGZ}
+
+run-standalone: setup
```

Comment:
Can we use a shorter name here? I always find long `make` targets hard to remember :( Or maybe it's not even worth having this as a target...

Reply:
I want to imply that we are running outside krun (there are two run modes).

From my POV, I don't care too much about the length of the target name. I just type `make run<tab>` and it completes. Even bash does this.

Reply:
I guess my question is also whether we want to tell people to use a `make` target or a shell script.

Reply:
I've opted to tell them to use the `Makefile` (also stopped `run_benchmarks.sh` calling back into `make`).

It seems simpler for a user to have one place to look. I find this *can* break down if the user frequently need to change command line arguments to things, but that doesn't seem to be the case here. The args seem fixed.

Reply:
OK.